### PR TITLE
Update scsi_da.c

### PR DESCRIPTION
**Pull request by mr44er**
my example: <SAMSUNG MZ7TE128HMGR-0SM 4321>    at scbus0 target 41 lun 0 (pass1,da0)

### CODE REVIEW

**Comment by mr44er** [comment]
Yup, thx.

File: sys/cam/scsi/scsi_da.c [hunk 1]
```diff
@@ -363,7 +363,7 @@ struct da_softc {
 	uint32_t		unmap_max_lba; /* Max LBAs in UNMAP req */
 	uint32_t		unmap_gran;
 	uint32_t		unmap_gran_align;
-	uint64_t		ws_max_blks;
+quirk pr	uint64_t		ws_max_blks;
```
Comment (bsdimp) [inline sys/cam/scsi/scsi_da.c:366]: 'quirk pr' is improper here and should be deleted.

**Comment by bsdimp** [comment]
I've merged this w/o the troublesome 'quirk pr' strays.

**Comment by bsdimp** [comment]
And sorry, but this totally slipped my mind... a lot has happened since Feb...

**Pull request closed without merging**